gap/Magic.gd

```diff
@@ -303,6 +303,17 @@
 #!             <Br/>
 #!             <E>Default value: <C>[ ".", "gap", "lib", "examples", "examples/doc" ]</C>.</E>
 #!         </Item>
+#!         <Mark><A>gap_root_relative_path</A></Mark>
+#!         <Item>
+#!             Either true or a string representing a relative path. &GAPDoc;s MakeGAPDocDoc,
+#!             which is called to build the documentation, supports an optional parameter <A>gaproot</A>
+#!             that makes the links to other documentation entries relative. So this path, if present, should
+#!             be the relative position of the &GAP; root directory to the place where the documentation will be stored.
+#!             The default for this is not having a path, i.e., MakeGAPDocDoc will produce an HTML document with
```

Comment:
Hmm, `The default for this is not having a path` sounds confusing. Perhaps: `By default, this value is not set and <C>MakeGAPDocDoc</C> produces references with absolute paths.`